packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -326,7 +326,6 @@ describe('Force Project Create', () => {
         'triggers'
       ];
       const filestocopy = [
-        '.eslintignore',
```

Comment:
same question here :)

Reply:
@CristiCanizales recall the change you made to templates that removed this from project sample files? Removing it here because our create project tests are verifying the file should be present. Removing here to match the work done in templates.